src/trackers/TVC.py

```diff
@@ -290,6 +320,21 @@ async def upload(self, meta, disctype):
             meta['tracker_status'][self.tracker]['status_message'] = "Debug mode enabled, not uploading."
         open_torrent.close()
 
+    def get_audio_languages(self, meta, mi):
+        """
+        Populate meta['audio_languages'] with normalized language names (Titlecase).
+        Synchronous because it parses the already-loaded MediaInfo JSON.
+        """
+        audio_langs = set()
+        for track in mi.get("media", {}).get("track", []):
+            if track.get("@type") == "Audio" and "Language" in track:
+                lang = str(track["Language"]).strip()
+                if lang:
+                    # Normalize to Titlecase so comparisons like "English" match
+                    audio_langs.add(lang.title())
+        meta['audio_languages'] = list(audio_langs) if audio_langs else []
```

Comment:
We should never update common meta objects from a site file. If the common meta object needs improvement, it should be modified in the original function..